packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -156,16 +156,16 @@ const NotificationDialog = (props) => {
   /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
   return (
     <AbstractModal
-      ariaLabel={signalWord}
-      aria-labelledby="notification-dialog-signal-word"
-      aria-describedby={dialogTitle ? 'notification-dialog-title' : 'notification-dialog-signal-word'}
+      ariaLabelledBy="notification-dialog-signal-word"
+      aria-describedby={dialogTitle ? 'notification-dialog-title' : undefined}
       role="alertdialog"
       classNameModal={classNames(cx('notification-dialog', theme.className), customProps.className)}
-      isOpen
       onRequestClose={() => {}}
-      closeOnEsc={false}
+      isOpen
+      closeOnEsc
```

Comment:
Do we still need this, since its true by default in AbstractModal ?

Reply:
Removed